h2o-core/src/main/java/water/util/ArrayUtils.java

```diff
@@ -559,6 +559,11 @@ public static String arrayToString(int[] ary) {
     return false;
   }
 
+  static public boolean contains(byte[] a, byte d) {
+    for (int anA : a) if (anA == d) return true;
```

Comment:
why do you cast to an `int`? Is that intentional?

Reply:
nope :) copy/pasted the int version, forgot this bit, thanks for catching this